app/Http/Controllers/Member/LoginController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers\Member;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\LoginRequest;
+use App\Models\Member;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+class LoginController extends Controller
+{
+    public function index() {
+        return view('member.mem-login');
+    }
+    public function login(LoginRequest $request) {
+        $credentials = $request->only('email' , 'password');
+        if(auth()->guard('member')->attempt($credentials ,  $request->filled('remember'))) {
```

Comment:
```suggestion
        if (auth()->guard('member')->attempt($credentials ,  $request->filled('remember'))) {
```

Dự án cá nhân thì ko quan trọng lắm, nhưng dự án mình đang sử dụng PHPCS nên những chỗ như này sẽ cần phải sửa lại, e chú ý nhé